rrd-lib.sh

```diff
@@ -33,6 +33,12 @@ get_devices () {
     DevTemp=$(/usr/local/sbin/smartctl -a /dev/"${i}" | awk -f $GETTEMP)
     if [ -n "$DevTemp" ]; then
       drivedevs="${drivedevs} ${i}"
+    else
+      #In case the disk is a SAS/SCSI disk without the default number pattern for Smart output
+      DevTemp=$(/usr/local/sbin/smartctl -a /dev/"${i}" | grep "Current Drive Temperature:" |awk '{print $4}')
+      if [ -n "$DevTemp" ]; then
+        drivedevs="${drivedevs} ${i}"
+      fi
       [ -n "$verbose" ] && echo "drivedevs: ${drivedevs}"
```

Comment:
This verbose/debug output (line 42) should occur after each `drivedevs` update. Could you place this verbose line after line 35 and after line 40?